.golangci.yml

```diff
@@ -3,30 +3,31 @@ linters-settings:
     locale: US
 
 linters:
-  # please, do not use `enable-all`: it's deprecated and will be removed soon.
-  # inverted configuration with `enable-all` and `disable` is not scalable during updates of golangci-lint
-  # disable-all: true
   enable:
     - golint
     - misspell
-
-  # don't enable:
-  # - gochecknoglobals
-  # - gocognit
-  # - godox
-  # - maligned
-  # - prealloc
+    - gocyclo
+    - bodyclose
+    - unconvert
+    - goconst
+    - goimports
+    - unparam
```

Comment:
Useful. Detected functions with received params that aren't used, which is a bit of a smell.
Also, detected a bunch of functions which returned `error` but the returned value was always `nil`.